### DISCUSSION TITLE
Error while deploying : Your Team encountered an unknown problem. Please reach out to our support team for details.

### DISCUSSION THREAD

**Discussion by praneethravuri** [vercel/next.js#64004] [Help]
### Summary

Error popping up while trying to deploy Nextjs application on Vercel
<img width="647" alt="Screenshot 2024-04-02 212356" src="https://github.com/vercel/next.js/assets/67636853/8659f42b-3b65-440b-8b07-fd6977e63aec">

### Additional information

I have a Nextjs application on GitHub which I have imported into Vercel. Whenever I try to deploy it, I am getting this error:

Your Team encountered an unknown problem. Please reach out to our support team for details.

I have created an account on Vercel for the first time and this is the first project I have been trying to deploy. Any help is appreciated.

**Comment by thuanbui-1203** [comment]
oh my god no answer and it has been a year already. I'm having the same problem and literally stuck atm
